lib/wash/exe/sort.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @typedef JsExecuteContext$$module$axiom$fs$js$execute_context */
+var JsExecuteContext;
+
+/**
+ * @param {JsExecuteContext} cx
+ * @return {void}
+ */
+export var main = function(cx) {
+  cx.ready();
+  // We might need to read from stdin: pause it so no data is lost.
+  cx.stdin.pause();
```

Comment:
This should be a no-op, as commands are guaranteed to receive paused readable streams (e.g. stdin), right?